markdown/api-docs/cart-and-checkout/embedded-checkout.md

```diff
@@ -0,0 +1,335 @@
+
+<!--
+title: "Create react app"
+subtitle: ""
+lineNumbers: true
+-->
+
+```shell
+npx create-react-app bc-embedded-checkout
+
+cd bc-embedded-checkout
+
+npm start
+```
+
+### BigCommerce Checkout SDK
+Install the BigCommerce Checkout SDK to invoke a method that can render the checkout in your app.
+
+In your project folder, run the following in your shell:
+
+<!--
+title: "Install Checkout SDK"
+subtitle: ""
+lineNumbers: true
+-->
+
+```shell
+npm install --save @bigcommerce/checkout-sdk
+
+```
+
+### Heroku
+
+We'll be deploying this app on [Heroku](https://heroku.com). Create a new app and give it a name. We will use the app's public URL to create a Channel in the next step.
+
+<a href='#embedded-checkout_create-channel' aria-hidden='true' class='block-anchor'  id='embedded-checkout_create-channel'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 2: Create a Channel
+
+In order to allow your external website to serve the BigCommerce checkout, you must first create a new Channel using the Channels API.
+
+To do this, make a POST request to `https://api.bigcommerce.com/stores/{{store_hash}}/V3/channels`
+
+<!--
+title: "POST to Channels"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Channel POST**
+
+```js
+{
+    "type": "storefront",
+    "platform": "custom",
+    "name": "https://bc-react.herokuapp.com"
+}
+
+```
+
+The response will be an `id` which you will use as the `channel_id` in future requests. Let's assume it's `15001` for this guide.
+
+Once you create a Channel, it appears in your BigCommerce Control Panel under Products > Listed On. Here products can be chosen to be listen on each desired channel. The Orders section will now also include a filter for your channel.
+
+<a href='#embedded-checkout_create-site' aria-hidden='true' class='block-anchor'  id='embedded-checkout_create-site'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 3: Create a Site
+
+Next you will need to create a site for the channel. Make a POST to `https://api.bigcommerce.com/stores/{{store_hash}}/v3/channels/{{channel_id}}/site`
+
+<!--
+title: "POST to Channels"
+subtitle: ""
+lineNumbers: true
+-->
+
+**Create Site POST**
+
+```js
+{
+    "channel_id": 15001,
+    "url": "https://bc-react.herokuapp.com"
+}
+
+```
+
+This returns `id` which you will use as the `site_id` in future requests. The `url` value is the base path for all other routes you define for the site.
+
+
+<a href='embedded-checkout_create-cart' aria-hidden='true' class='block-anchor'  id='embedded-checkout_create-cart'><i aria-hidden='true' class='linkify icon'></i></a>
+
+## 4: Create a Cart
+
+In order to proceed to checkout, an active cart is required. You'll use the [Server-to-Server Cart API](https://developer.bigcommerce.com/api-reference/cart-checkout/server-server-cart-api) to generate it.
```

Comment:
There should be an example request here showing how to get products and make a cart via API. There's probably an existing example in another article you can use. You can mention that the product IDs used in the example will need to be replaced, or do something like this in the request json: 

```json
{
product_id: <your_products_id>
}
```